client/styles/sign-up.css

```diff
@@ -1,18 +1,34 @@
-.form__title {
-   color: #2f195f;
-   font-size: 2opx;
-   text-align: center;
+.sign-page {
+   background-color: #f7f7fc;
+   display: flex;
+   position: absolute;
+   flex-direction: row;
+   justify-content: space-between;
+   align-items: center;
+   box-sizing: border-box;
+   margin: 0;
+   border: 0;
+   padding: 0;
+   top: 0;
+   right: 0;
+   bottom: 0;
+   left: 0;
+   min-height: 100vh;
+   z-index: -1;
 }
 
-.form-container {
+/* FORM SECTION */
+
+.form-registration {
```

Comment:
This section is not unique to this component. Maybe it would be better to add a signup) prefix
eg signup-form-registration .....